apps/x509.c

```diff
@@ -568,14 +568,8 @@ int x509_main(int argc, char **argv)
 
     if (reqfile) {
         EVP_PKEY *pkey;
-        BIO *in;
-
-        in = bio_open_default(infile, 'r', informat);
-        if (in == NULL)
-            goto end;
-        req = PEM_read_bio_X509_REQ(in, NULL, NULL, NULL);
-        BIO_free(in);
 
+        req = load_csr(infile, informat, "certificate request input");
```

Comment:
The ERR_print_errors() below seems to be superfluous as you're already calling it in load_csr().

Reply:
Thanks @t8m for your comment.
I've cleaned up the use of `ERR_print_errors()` also in four other apps and in `apps/lib/apps.c` (and made sure that all error message there consistently begin upper-case).